react/src/components/AssetDetail/AssetButton.tsx

```diff
@@ -19,27 +25,37 @@ const AssetButton: React.FC<AssetButtonProps> = ({
   onQuestionnaireClick,
 }) => {
   const pointCloudURL = DOMPurify.sanitize(featureSource + '/index.html');
-
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
   const featureType = getFeatureType(selectedFeature);
   const projectId = selectedFeature.project_id;
+  /* Uncomment when WG-422 polling is merged 
+  const notification = useNotification();*/
   const featureId = selectedFeature.id;
-  const {
-    mutate: importFeatureAsset,
-    isPending: isImporting,
-    isSuccess: isImportingSuccess,
-  } = useImportFeatureAsset(projectId, featureId);
+  const { mutate: importFeatureAsset, isPending: isImporting } =
+    useImportFeatureAsset(projectId, featureId);
 
-  const handleImportFeatureAsset = (importData: IFileImportRequest) => {
-    importFeatureAsset(importData);
-  };
-  const handleSubmit = () => {
-    const importData: IFileImportRequest = {
-      /*TODO Replace with passed in values from
-      FileBrowserModal. These are hardcoded to test.*/
-      system_id: 'project-4072868216578445806-242ac117-0001-012',
-      path: 'images_good/image.jpg',
-    };
-    handleImportFeatureAsset(importData);
+  const handleSubmit = (files: TapisFilePath[]) => {
+    for (const file of files) {
+      const importData: IFileImportRequest = {
+        system_id: file.system,
+        path: file.path,
+      };
+      importFeatureAsset(importData, {
+        onSuccess: () => {
+          setIsModalOpen(false);
+          /*Uncomment these notifications when WG-422 is merged
+          notification.success({
+            description: `Your asset was successful imported for feature ${selectedFeature.id}`,
```

Comment:
The import will take longer as the POST just starts/queues a celery task to do the work.  So text should be something like "Import started!"

Reply:
i got this wrong. i thought this endpoint starte a celery task.